src/metabase/driver/interface.clj

```diff
@@ -1,14 +1,4 @@
-(ns metabase.driver.interface
-  "Protocols that DB drivers implement. Thus, the interface such drivers provide."
-  (:import (clojure.lang Keyword)))
-
-(def ^:const driver-optional-features
-  "A set on optional features (as keywords) that may or may not be supported by individual drivers."
-  #{:foreign-keys
-    :nested-fields                         ; are nested Fields (i.e., Mongo-style nested keys) supported?
-    :set-timezone
-    :standard-deviation-aggregations
-    :unix-timestamp-special-type-fields})
+(ns metabase.driver.interface)
```

Comment:
Yeah. Part of the reason it was in a separate namespace in the first place was preventing circular dependencies that happened when we needed to import the protocols and what-not. Now that we're moving to an informal spec that's not an issue anymore